modules/apis/src/main/java/com/axway/apim/api/export/ExportAPI.java

```diff
@@ -74,6 +74,9 @@ public APISpecification getAPIDefinition() {
 		while(it.hasNext()) {
 			OutboundProfile profile = it.next();
 			profile.setApiId(null);
+			if("_default".equals(profile.getAuthenticationProfile())) {
+				profile.setAuthenticationProfile(null);
+			}
```

Comment:
Also here add a one-line comment why this is required to do.

Reply:
I don't think it was one of my commits

Reply:
Yes. You are right!

Reply:
Added comment